test/functional/pdo_sqlsrv/pdo_019_next_result_set_pooling.phpt

```diff
@@ -1,61 +1,90 @@
 --TEST--
 Moves the cursor to the next result set with pooling enabled
 --SKIPIF--
```

Comment:
It probably doesn't matter but we should still test it.

Are you referring to your comments about ORDER BY in pdo_018? I only change the SQL string between AE and non AE because I know a feature is not support in encrypted columns, and I've tried to reproduce the same result as non AE as much as possible.

Reply:
again please justify your changes by putting a comment / references

Reply:
ok